charabia/Cargo.toml

```diff
@@ -40,7 +40,10 @@ zerovec = "0.10.1"
 default = ["chinese", "hebrew", "japanese", "thai", "korean", "greek", "latin-camelcase", "latin-snakecase", "khmer", "vietnamese"]
 
 # allow chinese specialized tokenization
-chinese = ["dep:pinyin", "dep:jieba-rs"]
+chinese = ["chinese-segmentation", "chinese-normalization"]
+chinese-segmentation = ["dep:jieba-rs"]
+chinese-normalization = []
+chinese-normalization-pinyin = ["dep:pinyin", "chinese-normalization"]
```

Comment:
Will you reflect that on Meilisearch, too?

Reply:
no, I don't think so, 
everybody complains about this normalizer but I keep it in case another user of charabia wants to keep it